actors/market/src/lib.rs

```diff
@@ -318,9 +322,11 @@ impl Actor {
                 info!("invalid deal: {}: insufficient client funds to cover proposal cost", di);
                 continue;
             }
-            total_provider_lockup += &deal.proposal.provider_collateral;
+
+            let mut provider_lockup = total_provider_lockup.clone();
+            provider_lockup += deal.proposal.provider_collateral.clone();
```

Comment:
```suggestion
            provider_lockup += &deal.proposal.provider_collateral;
```

(should work? I think?)

Reply:
yes!